DD 4.15: Dev Velocity and POS content updates (#16176)
docker/docs: release-lifecycle.md

| --- | --- | ---- | ---| ---|
|[Experimental](#experimental)| Limited availability | Community support |Software may have limitations, bugs and/or stability concerns | Can be discontinued without notice |
|[Beta](#beta) | All or those involved in a Beta Feedback Program | Community support | Software may have limitations, bugs and/or stability concerns | Can be discontinued without notice |
| [Early Access (EA)](#early-access-ea) | All or those involved in an Early Access Feedback Program | Full | Software may have limitations, bugs and/or stability concerns.  These limitations will be documented. | Follows [retirement process](#retirement-process) |
| [General Availability (GA)](#general-availability-ga) | All | Full | Few or no limitations for supported use cases | Follows [retirement process](#retirement-process) |

### Experimental

Experimental offerings are features that Docker is currently experimenting with. Customers who access experimental features have the opportunity to test, validate, and provide feedback on future functionality. This helps us focus our efforts on what provides the most value to our customers.

> **Customer availability:** Availability of experimental features is limited. A portion users may be opted into none, one or many experimental features. 

Reviewer: ```suggestion
**Customer availability:** Availability of experimental features is limited. A portion of users may have access to none, one or many experimental features. 
```
